### ISSUE TITLE
If you are looking for node bindings for an up to date version (9.5.0)

### ISSUE THREAD

**Issue by randing89** [zone117x/node-multi-hashing#55]
Check out our fork. https://github.com/bbscoin/node-multi-hashing

**Comment by zone117x** [comment]
Is your module backwards compatible? Would love to pull your code into here. If you are interested in becoming the maintainer of this repo let me know.

**Comment by randing89** [comment]
No. The new code won't be able to be compiled with the old node version.
This repo is being used by many projects and they are all running on node 0.15. Merging it back to here I afraid would be a breaking change for a lot of people.
